tests/Unit/Actions/BacklogData/GetBacklogDataActionTest.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Tests\Unit;
+
+use App\Models\User;
+use DateTime;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Illuminate\Support\Facades\Auth;
+use Tests\TestCase;
+use Xguard\LaravelKanban\Actions\BacklogData\GetBacklogDataAction;
+use Xguard\LaravelKanban\Models\Board;
+use Xguard\LaravelKanban\Models\Column;
+use Xguard\LaravelKanban\Models\Employee;
+use Xguard\LaravelKanban\Models\Row;
+use Xguard\LaravelKanban\Models\Task;
+
+class GetBacklogDataActionTest extends TestCase
+{
```

Comment:
You could use random numbers instead of hardcoded values and test other stuff, like what happens to shift outside the window and on the time window boundary